applicationLibrary/molecularDynamics/molecularDynamicsLibrary/ParticlePropertiesLibrary.h

```diff
@@ -341,7 +420,8 @@ void ParticlePropertiesLibrary<floatType, intType>::calculateMixingCoefficients(
         "ParticlePropertiesLibrary::calculateMixingCoefficients was called without any site types being registered!");
   }
 
-  _computedMixingData.resize(_numRegisteredSiteTypes * _numRegisteredSiteTypes);
+  _computedLJMixingData.resize(_numRegisteredSiteTypes * _numRegisteredSiteTypes);
+  _computedATMixingData.resize(_numRegisteredSiteTypes * _numRegisteredSiteTypes * _numRegisteredSiteTypes);
```

Comment:
As far as I can see `getNu()` uses `operator[]`, which does NOT throw when accessing an index out of bounds. It MIGHT trigger a segfault, but in general ["Accessing a nonexistent element through this operator is undefined behavior"](https://en.cppreference.com/w/cpp/container/vector/operator_at).
In this case, however, I think this is fine, because this seems like intentional misuse and the other option is to have a check in every call to `getNu()` which might become unnecessarily expensive in the functor.